__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data')
+
+afterAll(()=> {
+    return db.end()
+});
+
+beforeEach(() => {
+   return seed({articleData, commentData, topicData, userData})
+})
+
+describe('GET /api/topics', ()=> {
+    test('200: responds with an array of topic objects', ()=> {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            response.body.topics.forEach((topic) => {
```

Comment:
To avoid a false positive in this test if you were passed an empty array, you must combine any loop through an array with a check of how long the array is. Please add an expect statement which checks the length of the topics array